compiler/src/iree/compiler/Dialect/HAL/Target/LLVM/LLVMCPUTarget.cpp

```diff
@@ -827,17 +827,23 @@ void registerLLVMAOTTargetBackends(
 #include "llvm/Config/Targets.def"
 
   auto backendFactory = [=]() {
-    return std::make_shared<LLVMAOTTargetBackend>(queryOptions());
+    return std::make_shared<LLVMTargetBackend>(queryOptions());
   };
 
+  // Preferred name.
+  // #hal.executable.target<"llvm-cpu", ...
+  static TargetBackendRegistration registration0("llvm-cpu", backendFactory);
+
+  // Abbreviated names.
   // #hal.device.target<"cpu", ...
-  static TargetBackendRegistration registration0("cpu", backendFactory);
+  static TargetBackendRegistration registration1("cpu", backendFactory);
```

Comment:
can we drop these? (or is this a followup?)

Reply:
Follow-up, yeah. I'll go through and update our tests/samples/etc. that use the old names. I wanted to keep this diff smaller and give people some time to switch their scripts / downstream code over.

Should we keep the `cpu` and `llvm` names as shorthands, or only use `llvm-cpu`?

Reply:
at least the `cpu` shorthand would be great.